README.md

```diff
@@ -29,7 +29,7 @@ This will run the automated build process, start up a webserver, and open the ap
 7. **Having issues?** See "Having Issues?" below.
 
 ##Initial Machine Setup
-1. **Install [Node 4.0.0 or greater](https://nodejs.org)** - (5.0 or greater is recommended for optimal build performance). Need to run multiple versions of Node? Use [nvm](https://github.com/creationix/nvm).
+1. **Install [Node 4.0.0 or greater upto Node 6.x.x](https://nodejs.org)** - (5.0 or greater upto 6.x is recommended for optimal build performance). Node 7 isn't supported yet because it has removed the harmony_proxies flag in favor of harmony. `--harmony_proxies` is a necessary flag for Node 3 and Node 4. Need to run multiple versions of Node? Use [nvm](https://github.com/creationix/nvm).
```

Comment:
May want to include Node 5. Travis builds failed without this for that version as well.
https://travis-ci.org/coryhouse/react-slingshot/builds/197890657